### DISCUSSION TITLE
next15 Latest version npm i@chakra-ui/React@emotion /react failed to be installed

### DISCUSSION THREAD

**Discussion by burnkings** [chakra-ui/chakra-ui#9109] [Q&A] [answered]
npm error code ERESOLVE
npm error ERESOLVE unable to resolve dependency tree
npm error
npm error While resolving: recruit_front@0.1.0
npm error Found: react@19.0.0-rc-02c0e824-20241028
npm error node_modules/react
npm error   react@"19.0.0-rc-02c0e824-20241028" from the root project
npm error
npm error Could not resolve dependency:
npm error peer react@">=16.8.0" from @emotion/react@11.13.3
npm error node_modules/@emotion/react
npm error   @emotion/react@"*" from the root project
npm error   peer @emotion/react@">=11" from @chakra-ui/react@3.1.0
npm error   node_modules/@chakra-ui/react
npm error     @chakra-ui/react@"*" from the root project
npm error
npm error Fix the upstream dependency conflict, or retry
npm error this command with --force or --legacy-peer-deps
npm error to accept an incorrect (and potentially broken) dependency resolution.

What's the problem

**Comment by veloware** [comment]
Im seeing the same, ended up using `--legacy-peer-deps`

**Reply by abhinav-lv** [reply]
It's because Next.js 15 has started to use React Canary builds for the App Router by default, but Chakra UI needs the latest stable build of react. I've been racking my brains to use a lower version of Next.js but everytime I try to install Chakra UI it just doesn't work. Someone please suggest a fix.

**Reply by burnkings** [reply]
npm i @chakra-ui/react @emotion/react --legacy-peer-deps

**Reply by abhinav-lv** [reply]
``` npx @chakra-ui/cli snippet add```

If I use `--legacy-peer-deps` I get all sorts of errors when I execute the above line to add components

**Reply by burnkings** [reply]
My node is 22.11, running npx@chakra-ui /cli snippet add cannot be installed properly

**Comment by segunadebayo** [comment, marked as answer]
Duplicate of https://github.com/chakra-ui/chakra-ui/discussions/9020

Same Next 15 / React 19 RC peer-dep conflict. Current `@chakra-ui/react` installs cleanly. Update the package.
